static/js/drafts.js

```diff
@@ -229,9 +233,15 @@ exports.format_draft = function (draft) {
         // In case there is no stream for the draft, we need a
         // single space char for proper rendering of the stream label
         const space_string = new Handlebars.SafeString("&nbsp;");
-        const stream = draft.stream.length > 0 ? draft.stream : space_string;
+        let stream = draft.stream.length > 0 ? draft.stream : space_string;
```

Comment:
Can we rename this to `stream_name` so that we're clear that this is not an object or an id or something else? The check against `draft.stream.length` hints that it's a string, but it wouldn't hurt to be explicit.